<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgParam</class>
 <widget class="QWidget" name="AlgParam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*groupBox*/
QGroupBox{
/*background:#444444;*/
border:1px solid gray;
border-radius:10px;
margin-top:11px;
}

QGroupBox::title {
      subcontrol-origin: margin;
      subcontrol-position: top left;
      padding-top: 5;
      padding-left: 5;
}
/*lineedit*/
QLineEdit {
        border: 1px solid #A0A0A0; /* 边框宽度为1px，颜色为#A0A0A0 */
        border-radius: 3px; /* 边框圆角 */
        padding-left: 5px; /* 文本距离左边界有5px */
        background-color: #F2F2F2; /* 背景颜色 */
        color: #A0A0A0; /* 文本颜色 */
        selection-background-color: #A0A0A0; /* 选中文本的背景颜色 */
        selection-color: #F2F2F2; /* 选中文本的颜色 */
        font-family: &quot;Microsoft YaHei&quot;; /* 文本字体族 */
        font-size: 10pt; /* 文本字体大小 */
}
/*pushbutton*/
QPushButton {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    padding: 5px;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #ecf5ff;
    color: #409eff;
}

QPushButton:pressed, QPushButton:checked {
    border: 1px solid #3a8ee6;
    color: #409eff;
}

QComboBox{
        border:1px solid gray;
        border-radius:3px;
        padding: 5px;
        min-width:4em;
}
QComboBox::drop-down{
subcontrol-origin:padding;
subcontrol-position:top right;
width:20px;
border-left-width:1px;
border-left-color:darkgray;
border-left-style:solid;
border-top-right-radius:3px;
border-bottom-right-radius:3px;
}
QComboBox QAbstractItemView{
border: 2px solid #4E6D8C;
}
QComboBox::down-arrow{
image: url(:/image/down.png);
}

/*表格美化*/
QHeaderView
{
    background:transparent;
}

QHeaderView::section
{
    font-size:14px;
    font-family:&quot;Microsoft YaHei&quot;;
    color:#FFFFFF;
    background:#60669B;
    border:none;
    text-align:left;
    min-height:49px;
    max-height:49px;
    margin-left:0px;
    padding-left:0px;
}

#tableWidget_SenceInfo
{
    background:#FFFFFF;
    border:none;

    font-size:10px;
    font-family:&quot;Microsoft YaHei&quot;;
    color:#666666;
}
QTableWidget::item
{
    border-bottom:1px solid #EEF1F7 ;
}

QTableWidget::item::selected
{
    color:red;
    background:#EFF4FF;
}


QScrollBar::handle:vertical
{
    background: rgba(255,255,255,20%);
    border: 0px solid grey;
    border-radius:3px;
    width: 8px;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
    background:rgba(255,255,255,10%);
}


QScollBar::add-line:vertical, QScrollBar::sub-line:vertical
{
    background:transparent;
}

/*QCheckBox*/
QCheckBox{
	border: none;/*最外层边框*/
}
QCheckBox::indicator{/*选择框尺寸*/
    background-color: rgba(255, 255, 255);
    border: 0px solid black;
	width: 30px;
	height: 30px;
}
QCheckBox:enabled:checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:!checked{
	color: rgb(0, 0, 0);
}
QCheckBox:enabled:hover{
 	color: rgb(0, 200, 0);
}
QCheckBox:!enabled{
	color: rgb(80, 80, 80);
}
 
QCheckBox::indicator:unchecked {
        image: url(:/image/Switch-close.png);
}
QCheckBox::indicator:unchecked:hover {
       image: url(:/image/Switch-close.png);
}
QCheckBox::indicator:unchecked:pressed {
        image: url(:/image/Switch-close.png);
}
QCheckBox::indicator:checked {
        image: url(:/image/Switch-open.png);
}
QCheckBox::indicator:checked:hover {
        image: url(:/image/Switch-open.png);
}
QCheckBox::indicator:checked:pressed {
        image: url(:/image/Switch-open.png);
}

/*treeWidget*/
QTreeView::branch:has-siblings:!adjoins-item {
	border-image: url(:/image/vline.png) 0;
}
 
QTreeView::branch:has-siblings:adjoins-item {
	border-image: url(:/image/branch-more.png) 0;
}
 
QTreeView::branch:!has-children:!has-siblings:adjoins-item {
	border-image: url(:/image/branch-end.png) 0;
}
 
QTreeView::branch:has-children:!has-siblings:closed,
QTreeView::branch:closed:has-children:has-siblings {
    border-image: none;
    image: url(:/image/branch-closed.png);
}
 
QTreeView::branch:open:has-children:!has-siblings,
QTreeView::branch:open:has-children:has-siblings  {
    border-image: none;
    image: url(:/image/branch-open.png);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>相机内参</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLineEdit" name="Edit_CameraIn11_fx">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string>fx</string>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
              <property name="accessibleDescription">
               <string/>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="inputMask">
               <string/>
              </property>
              <property name="placeholderText">
               <string>fx</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="Edit_CameraIn12">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="Edit_CameraIn13_cx">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Cx</string>
              </property>
              <property name="placeholderText">
               <string>Cx</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="Edit_CameraIn21">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="Edit_CameraIn22_fy">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string>fy</string>
              </property>
              <property name="placeholderText">
               <string>fy</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="Edit_CameraIn23_cy">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Cy</string>
              </property>
              <property name="placeholderText">
               <string>Cy</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLineEdit" name="Edit_CameraIn31">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="Edit_CameraIn32">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="Edit_CameraIn33">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>外参矩阵</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>248</width>
            <height>54</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>旋转矩阵</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="Edit_RotateX">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Rx</string>
             </property>
             <property name="placeholderText">
              <string>Rx</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="Edit_RotateY">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Ry</string>
             </property>
             <property name="placeholderText">
              <string>Ry</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="Edit_RotateZ">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Rz</string>
             </property>
             <property name="placeholderText">
              <string>Rz</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>平移向量</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="Edit_TranslationX">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Tx</string>
             </property>
             <property name="placeholderText">
              <string>Tx</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="Edit_TranslationY">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Ty</string>
             </property>
             <property name="placeholderText">
              <string>Ty</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="Edit_TranslationZ">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Tz</string>
             </property>
             <property name="placeholderText">
              <string>Tz</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>畸变系数</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>248</width>
            <height>54</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QLineEdit" name="Edit_Distortion_k1">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>K1</string>
             </property>
             <property name="placeholderText">
              <string>K1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="Edit_Distortion_k2">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>K2</string>
             </property>
             <property name="placeholderText">
              <string>K2</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="Edit_Distortion_k3">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>K3</string>
             </property>
             <property name="placeholderText">
              <string>K3</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="Edit_Distortion_p1">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>P1</string>
             </property>
             <property name="placeholderText">
              <string>P1</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="Edit_Distortion_p2">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>P2</string>
             </property>
             <property name="placeholderText">
              <string>P2</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>280</width>
         <height>106</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>毫米波单应性矩阵</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLineEdit" name="Edit_RadarIn_h11">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>h11</string>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="accessibleDescription">
             <string/>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="placeholderText">
             <string>h11</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="Edit_RadarIn_h12">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>h12</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>h12</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="Edit_RadarIn_h13">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>h13</string>
            </property>
            <property name="placeholderText">
             <string>h13</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="Edit_RadarIn_h21">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>h21</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>h21</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="Edit_RadarIn_h22">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>h22</string>
            </property>
            <property name="placeholderText">
             <string>h22</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="Edit_RadarIn_h23">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>h23</string>
            </property>
            <property name="placeholderText">
             <string>h23</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="Edit_RadarIn_h31">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>h31</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>h31</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="Edit_RadarIn_h32">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>h32</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>h32</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="Edit_RadarIn_h33">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>h33</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>h33</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="Btn_ReadParam">
       <property name="text">
        <string>  读取</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_WriteParam">
       <property name="text">
        <string>写入</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="Label_Msg">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
